### ISSUE TITLE
implicit_figures caption should be the image title text, not the image alt text.

### ISSUE THREAD

**Issue by datagrok** [jgm/pandoc#1679]
The implicit_figures extension currently uses the image’s alt text as its caption. I think it should use the title text, instead.

alt text is used in HTML to describe the visual appearance of an image, including any text rendered in the image, for screen readers and text-mode browsers. This is not always appropriate for use in a caption, nor is text used in a caption necessarily meaningful as replacement text for images. Consider:

```
![Disk usage. A pie chart showing used=25%, free=75%](du.png "Our algorithm results in significant savings.")
```

Also, though this is less of an issue, I think the markdown reader should swap its notion of title and alt text. Sometimes it is appropriate to set alt="", and there's no way to do so when using image references.

```
![foo]
...
[foo]: /images/foo.jpg "I wish screen readers wouldn't say 'foo' here." 
```
